packages/flutter/lib/src/material/material_localizations.dart

```diff
@@ -422,30 +422,6 @@ abstract class MaterialLocalizations {
   /// shows the list of accounts.
   String get showAccountsLabel;
 
-  /// The semantics label used for [ReorderableListView] to reorder an item in the
-  /// list to the start of the list.
-  String get reorderItemToStart;
```

Comment:
If someone is using it, then yes. It is hard to imagine how people will reuse this though.

I am not sure whether people would extend this class. @thkim1011, do you know if there is use case for that?

Reply:
This is technically not a breaking change given all tests pass. The breaking change guide states it would be optional whether we announce this as a courtesy. I personally think it is very unlikely this would be a breaking change. WDYT?

Reply:
If no tests broke that sounds fine to me. I would probably do a global test run in Google before this lands though to make sure.